jeffwu0724/liferay-portal
# LPS-149258

Hey Tina, this is what I did in LPS-149258. I have attach the error log in this [google doc](https://docs.google.com/document/d/18ny2hj2vRa-tRvoPSfu7zTv9CLmCLz_WiU2ivU4QHAA/edit). 
How to reproduce locally:
1. start the portal and wait till it launch successfully. 
2. turn off the portal
3. clear the database as well as master/osgi/state and master/data
4. restart the portal, this error log should pop up

I basically move poi to shared-dependencies, and added dependencies to different bundles to pass the failing integration tests and unit tests found by ci. I used compileInclude instead of compileOnly in portal-tika because it will fail `TextExtractorTest` without doing so.

Thanks for checking!
Jeff
